src/net_processing.cpp

```diff
@@ -1519,6 +1519,12 @@ bool static ProcessMessage(CNode* pfrom, string strCommand, CDataStream& vRecv,
         CBlock block;
         assert(ReadBlockFromDisk(block, it->second, chainparams.GetConsensus()));
 
+        if (req.indexes.size() >= block.vtx.size()) {
+            LogPrintf("recv getblocktxn. too many requests (%d >= %d) peer=%d\n", req.indexes.size(), block.vtx.size(), pfrom->id);
```

Comment:
Please try to keep the style of logging consistent, see `LogPrintf` a few lines below.